addons/website/static/tests/tours/media_dialog.js

```diff
@@ -3,13 +3,13 @@
 import tour from 'web_tour.tour';
 import wTourUtils from 'website.tour_utils';
 
+// TODO in master: use registerWebsitePreviewTour with edition: true and remove
+// clickOnEditAndWaitEditMode.
 tour.register('website_media_dialog_undraw', {
     test: true,
     url: '/',
 }, [
-{
-    trigger: 'a[data-action=edit]',
-},
+...wTourUtils.clickOnEditAndWaitEditMode(),
```

Comment:
I made it that way to be stable safe. I suppose it would be to risky to replace tour.register by registerWebsitePreviewTour and remove th web_tour.tour import :thinking: